qo.gemspec

```diff
@@ -29,6 +29,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "yard"
   spec.add_development_dependency "redcarpet"
   spec.add_development_dependency "dry-matcher"
+  spec.add_development_dependency "trace_spy"
 
   spec.add_runtime_dependency "any", '0.1.0'
+  spec.add_runtime_dependency "parser"
```

Comment:
Need to remove this. Was trying to add some black magic for a "compile" step, turns out it won't work. Might revisit this later with `TraceSpy` or `TracePoint` and write an article on that later. The short version of it is you can intercept block calls and likely get the argument values.

This will give two potential things: deep destructuring, and dynamic compilation. It'll probably be a later project for shenanigans and fun.